anime_downloader/config.py

```diff
@@ -20,6 +20,9 @@
         'provider': 'twist.moe',
         'external_downloader': '',
         'aria2c_for_torrents': False,
+        'selescrape_browser': ".",
```

Comment:
I don't like using '.' as default. Why not 'default' or `None`

Reply:
well, that can be done, i just got inspired by the default value for the download directory.
The . (dot) that means the current working directory

Reply:
why then not 'currentWorkingDirectory', make all variables / comments explicit so anyone can understand immediately.